core/src/main/resources/sql.postgres/ddl.sql

```diff
@@ -0,0 +1,29 @@
+DROP TABLE certificate_tag;
+DROP TABLE certificate;
+DROP TABLE tag;
+
+CREATE TABLE certificate(
+    id               BIGSERIAL     NOT NULL PRIMARY KEY,
+    name             VARCHAR(150)  NOT NULL,
+    description      VARCHAR(255)  NOT NULL,
+    price            DECIMAL(5, 2) NOT NULL,
+    duration         INT           NOT NULL,
+    create_date      TIMESTAMP     NOT NULL,
+    last_update_date TIMESTAMP     NOT NULL
+);
+
+CREATE TABLE tag
+(
+    id   BIGSERIAL   NOT NULL PRIMARY KEY,
+    name VARCHAR(50) NOT NULL,
+    UNIQUE (name)
+);
+
+CREATE INDEX tag_name_case_insensitive_unique_index ON tag (LOWER(name));
+
+CREATE TABLE certificate_tag(
+    id             BIGSERIAL,
+    certificate_id BIGINT NOT NULL REFERENCES certificate (id),
```

Comment:
Remove id and make composite PK from certificate_id and tag_id

Reply:
done

Reply:
done(only table modify, now)